ci/run.sh

```diff
@@ -45,7 +45,7 @@ fi
 
 # Runs ctest to verify mach's ABI against the system libraries:
 if [[ -z "$NOCTEST" ]]; then
-    if [[ $TRAVIS_RUST_VERSION == "beta" ]] || [[ $TRAVIS_RUST_VERSION == "nightly" ]]; then
+    if [[ $TRAVIS_RUST_VERSION == "nightly" ]]; then
```

Comment:
The `ctest` now only pass on `nightly` because they require `repr(packed)` to pass. Before they used to pass on `beta` as well because we just ignored the undefined behavior on packed structs. I think it is more important for CI to catch these issues unconditionally than to just check beta.